packages/itwinui-css/src/slider/slider.scss

```diff
@@ -2,251 +2,256 @@
-      cursor: not-allowed;
-      pointer-events: none;
+  &[data-iui-orientation='horizontal'] {
+    $slider-position-horizontal: 0 var(--iui-slider-thumb-position);
+
+    grid-template-columns: auto 1fr auto;
+    grid-template-areas: 'min content max';
+    align-items: center;
+
+    .iui-slider-min,
+    .iui-slider-max {
+      align-items: center;
     }
 
-    .iui-slider-rail {
-      background-color: var(--iui-color-background-disabled);
+    .iui-slider-min {
+      margin-inline-end: $iui-slider-container-gap;
+      justify-content: end;
     }
 
-    .iui-slider-thumb {
-      cursor: not-allowed;
-      background-color: var(--iui-color-border);
-      border-color: var(--iui-color-border);
-      @media (forced-colors: active) {
-        background-color: GrayText;
-        border-color: GrayText;
+    .iui-slider-max {
+      margin-inline-start: $iui-slider-container-gap;
+    }
+
+    .iui-slider-ticks {
+      transform: translateY(50%);
+    }
+
+    .iui-slider-tick {
+      flex-direction: column;
+      inline-size: $iui-slider-tick-thickness;
+
+      &::before {
+        inline-size: $iui-slider-tick-thickness;
+        block-size: $iui-slider-tick-size;
       }
     }
 
-    .iui-slider-track {
-      background-color: var(--iui-color-border);
-      @media (forced-colors: active) {
-        background-color: GrayText;
+    .iui-slider {
+      block-size: calc($iui-slider-thumb-size + 2px);
+      touch-action: pan-x;
+
+      &::before {
+        inline-size: 100%;
       }
     }
 
+    .iui-slider::before,
+    .iui-slider-track {
+      block-size: var(--_iui-slider-track-thickness);
+    }
+
+    .iui-slider-track {
+      inset-inline: var(--iui-slider-track-position, #{$slider-position-horizontal});
+    }
+
+    .iui-slider-thumb {
+      inset-inline-start: var(--iui-slider-thumb-position);
+      transform: translateX(-50%);
+    }
+  }
+
+  &[data-iui-orientation='vertical'] {
+    $slider-position-vertical: var(--iui-slider-thumb-position) 0;
+
+    grid-template-rows: auto 1fr auto;
+    grid-template-areas:
+      'max'
+      'content'
+      'min';
+    justify-items: center;
+    inline-size: fit-content;
+
     .iui-slider-min,
-    .iui-slider-max,
+    .iui-slider-max {
+      justify-content: center;
+    }
+
+    .iui-slider-min {
+      margin-block-start: $iui-slider-container-gap;
+    }
+
+    .iui-slider-max {
+      margin-block-end: $iui-slider-container-gap;
+    }
+
+    .iui-slider-ticks {
+      flex-direction: column-reverse;
+      transform: translateX(50%);
+    }
+
     .iui-slider-tick {
-      user-select: none;
-      color: var(--iui-color-text-disabled);
-      @media (forced-colors: active) {
-        color: GrayText;
+      block-size: $iui-slider-tick-thickness;
+
+      &::before {
+        inline-size: $iui-slider-tick-size;
+        block-size: $iui-slider-tick-thickness;
+      }
+    }
+
+    .iui-slider {
+      inline-size: calc($iui-slider-thumb-size + 2px);
```

Comment:
Just wondering, what is the `2px` for?

Reply:
`$iui-slider-thumb-size: var(--iui-size-m);` & size m = 16px.

If I inspect the thumb, I see 18x18:
![Screenshot 2023-10-16 at 2 23 19 PM](https://github.com/iTwin/iTwinUI/assets/849817/17d994ea-783e-4cc6-bb65-132870458d84)

So I think it must be the size of the thumb + the border thickness.

Maybe it should be this instead?
```scss
$iui-slider-thumb-border-thickness: 1px;

.iui-slider {
  inline-size: calc($iui-slider-thumb-size + ($iui-slider-thumb-border-thickness * 2));
}
```